R/showToast.R

```diff
@@ -1,5 +1,21 @@
 
-
+#' @noRd default_options
+default_options <- list(
```

Comment:
I would change the name of this to `showToast_default_options`.  `default_options` is a little too generic.

Reply:
9980df064c237c4fb83b8d87de40b609b04894a3